iiify/templates/docs/overview.html

```diff
@@ -1,242 +1,94 @@
+
+      <p>The Presentation API can also faciliate the portability and playback of time-based media like audio and video files. In the A/V context, IIIF manifests often present multiple playback options based on the media formats and quality types available.</p>
+      
+      <p>A manifest for an Internet Archive item looks like this:</p>
+    <div class="request">https://iiif.archive.org/iiif/3/<b>:id</b>/manifest.json</div>
 
-    <div class="request">
-      GET {{ request.url_root }}
-    </div>
+          <p>see the sections below for all the possible ways to retrieve manifests for <a href="#items">items</a> and <a href="#collections">collections</a>.</p>
 
-    <h3>Detailed Representations</h3>
+<h2><a name="image-api"></a>Image API</h2>
     <p>
-      To get detailed metadata about a specific Archive.org item, fetch
-      the item's metadata by ID through the Archive.org. (Note that
-      authorization sometimes influences the amount of detail included
-      in the representation.)
+      The <a href="http://iiif.io/api/image/">Image API</a> is an
+      interface capable of producing and serving arbitrary tiles of
+      arbitrary regions of an image (just like the different scales of resolution in Google Maps). The properties
+      of these image tiles are specified within a URL, allowing
+      permenant citation and retrieval at a later time.
+      The Image API supports and powers other computer applications
+      (like <a href="https://universalviewer.io/">deep-zoomable image
+  viewers</a>, <a href="https://projectmirador.org/">comparison viewers</a>, and <a href="https://iiif.io/get-started/iiif-viewers/">more</a>).
     </p>
 
-    <div class="request">
-      GET https://archive.org/<strong>metadata</strong>/:id
-    </div>
-
-    <p>For example:</p>
-    <pre><code>
-  $ curl -i "https://archive.org/metadata/thirteenbookseu03heibgoog"
-
-  HTTP/1.1 200 OK
-  Server: nginx/1.4.6 (Ubuntu)
-  Date: Tue, 22 Sep 2015 08:07:27 GMT
-  Content-Type: application/json
-  Content-Length: 21307
-  Connection: keep-alive
-  Cache-Control: max-age=5
-
-  {
-    created: 1442378110,
-    d1: "ia600309.us.archive.org",
-    ad2: "ia800309.us.archive.org",
-    dir: "/20/items/thirteenbookseu03heibgoog",
-    files: [],
-    files_count: 16,
-    item_size: 196609584,
-    metadata: {},
-    reviews: [],
-    server: "ia800309.us.archive.org",
-    uniq: 893310999,
-    updated: 1442380951,
-    workable_servers: []
-  }
-    </code></pre>
+    <p>The Image API creates images using the following template: </p>
+      <div class="request">{scheme}://{server}/{prefix}/{identifier}/{region}/{size}/{rotation}/{quality}.{format}</div>
+    <p>In the context of the Internet Archive items, that will look like this for a full image:</p>
+    <div class="request">https://iiif.archive.org/image/iiif/3/<b>:id+filename</b>/full/max/0/default.jpg</div>
```

Comment:
Could we add a comment to say the id+filename combination can be got from the download link? For example if the file download link is:

https://archive.org/download/img-8664_202009/IMG_8664.jpg

then the IIIF image API URL would be (with / encoded as %2f):

https://iiif.archive.org/image/iiif/3/img-8664_202009%2fIMG_8664.jpg

I know it adds complexcity to the description but its an implementation detail that can easily be forgotten and if people are looking at how to genereate the Image API then knowing how it works will be useful.

Reply:
Ok, I've put in a few lines about this, thanks Glen